assets/lang/sklmessages_en_US.properties

```diff
@@ -40,7 +40,7 @@ game.output.type.text=Text
 
 game.conflict.title=Duplicate instance warning
```

Comment:
I'm not sure, but it seems to me that this sentence should be capitalized. I can correct this, if needed.

---
BTW, are `settings.lang.help` and `settings.lang.link` still needed in this file?

Reply:
Yes, capitalization here would make more sense.

And no, I think it's not needed atm, but I would like to add something like Discord have.
![image](https://user-images.githubusercontent.com/3246162/43274831-855dd1b6-9100-11e8-8216-e5fef1e26787.png)

Reply:
OK. So I'll make a fix for this. Just a moment.

Reply:
This sounds nice. The bad thing in these two strings is that they are only displayed if U.S. English language is selected. So if a new translator (like TrueEL or NightFurySL2001) see these two strings, he will translate them, wasting time, because they will not be displayed anyway.
My first idea was to hard-code these two strings into the launcher source code, and remove them from all lang files. But the Discord's idea sounds better.

Reply:
Description in `game.conflict.title` has been fixed.